[RM Ch02] 6G Related Use Cases (#3253)
anuket-project/anuket-specifications: doc/ref_model/chapters/chapter02.rst

hosted at the Edge. The Telco Edge is a perfect fit for centralized vRAN deployment and vDU/vCU hosting that satisfy the
latency requirements.

> It is expected that with the technology evolution (e.g. 6G) the use cases will be more demanding. For instance, either to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required to evolve the architecture. These use cases, once available, can be used for life saving decisions, for instance for the remote automation in environments not supporting life (e.g., in deep space communication), to ensure that the car autonomous driving can be done in real time, and even for holographic communications. Such use cases can be seen as the evolution of 5G use cases, where such requirements could not be met due to the technology constrains.

Reviewer: Please wrap lines to 120 chars.

> It is expected that with the technology evolution (e.g. 6G) the use cases will be more demanding. For instance, either to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required to evolve the architecture. These use cases, once available, can be used for life saving decisions, for instance for the remote automation in environments not supporting life (e.g., in deep space communication), to ensure that the car autonomous driving can be done in real time, and even for holographic communications. Such use cases can be seen as the evolution of 5G use cases, where such requirements could not be met due to the technology constrains.

Author: ```suggestion
It is expected that with the technology evolution (e.g. 6G) the use cases will be more demanding. For instance, 
either to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required to evolve the architecture. 
These use cases, once available, can be used for life saving decisions, for instance for the remote automation in 
environments not supporting life (e.g., in deep space communication), to ensure that the car autonomous 
driving can be done in real time, and even for holographic communications. Such use cases can be seen as the 
evolution of 5G use cases, where such requirements could not be met due to the technology constrains.
```

> It's expected with technology evolution (e.g. 6G) that use cases will be more demanding. For instance either  to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required an architectural evolution. These use cases once available can take action for life saving decisions, for instance remote automation on environments that life is not supported (e.g. deep space communication), car autonomous driving can be done in real time, and even holographic communications. Such use cases, which can be seen as an evolution of 5G Use Cases but that were not possible to be desgined due to the constrains created by the respective requirements, can now be considered.

Reviewer 2: ```suggestion
It is expected that with the technology evolution (e.g. 6G) the use cases will be more demanding. For instance, either to meet less than 1 ms latency, or ultra ultra fast data rate, it will be required to evolve the architecture. These use cases, once available, can be used for life saving decisions, for instance for the remote automation in environments not supporting life (e.g., in deep space communication), to ensure that the car autonomous driving can be done in real time, and even for holographic communications. Such use cases can be seen as the evolved examples of 5G use cases, where such requirements could not be met due to the technology constrains.
```

    - Distibuted deployment model across the ecosystem. It should be possbile to deploy workload at the extreme edge, which would allow realtime processing for video, and offload processing for network load prediction, which would support the Quality of Experience tat is defined for such a use case
    - The use case covers should allow the placement at the management plane and control plane (e.g. Core, Edge domain) 
    - There are high-leve requirements requirement for such a use case (e.g. latency of 1ms, available bandwidth 8 Gps 
      
- **Use Case #6 - Digital Twins for Manufacturing**

  - **Business Objectives**

    Providing the capability to design and create a product/service as a Digital Twin which can be tested before moving in to Production environment. Therefore, only once acceptance is achieved in the digital world, the service would be available. This leads to extreme reduction of Total Cost of Ownership (TCO), and minimize the risks that are comonly associated to a design and testing of a service for industrial enviroment.

  - **Targeted Segments**

    - Private Networks
    - Enterprise
    - Factory (make use of high level of automation).
    

>      - **Architecture**
> 
>     - Demands very low latency (<<1ms) and high reliabilit. 
>     - Trustworthiness needs to be guaranteed, which are usually associated to performance, security and resource efficiency/cost and subsequently productivity
>     - Processing cabaility of massive volumes of data.
>     

Reviewer 2: ```suggestion
   - **Architecture**

    - Demands very low latency (<<1ms) and high reliabilit. 
    - Trustworthiness needs to be guaranteed, which are usually associated to performance, security and resource efficiency/cost and subsequently productivity
    - Processing cabaility of massive volumes of data.
    
```

Reviewer 3: nit: reliabilit -> reliability
